harp/reader.py

```diff
@@ -214,7 +239,7 @@ def create_reader(
     keep_type: bool = False,
 ):
     """Creates a device reader object from the specified dataset or schema.
-    
+
```

Comment:
Is this formatting change a result of some automatic linter? I have been using `black` for all changes and it never touched these comments. I would rather keep formatting consistent with the rest of the code base.

Actually, do you mind running `black` on this file before we merge the PR? I think some of the lines earlier might not be following the style guidelines.